edx_notifications/server/api/consumer.py

```diff
@@ -117,7 +118,7 @@ def get(self, request):
             return Response({}, status.HTTP_400_BAD_REQUEST)
 
         cnt = get_notifications_count_for_user(
-            request.user.id,
+            int(request.user.id),
```

Comment:
Once I got this running in a devstack (as opposed to localdev running sqlite) it appears that request.user.id is a long.

I didn't see a way to have a @contract() which allowed for multiple types. Do you have an idea?